packages/teleport/src/Support/Support.story.tsx

```diff
@@ -18,6 +18,7 @@ import React from 'react';
 import Support from './Support';
 import * as teleport from 'teleport';
 import cfg from 'teleport/config';
+import { userContext } from 'teleport/Main/fixtures';
```

Comment:
this dependency on the Main fixtures looks strange because Main story would reach out to its children component for the same. This cross referencing seems unnecessary. Also, what's special about userContext that it needs to be imported from the fixtures to render a basic Support screen?

Reply:
refactored Support so fixtures and context wrappers are no longer needed